conf/messages

```diff
@@ -3977,9 +3977,9 @@ repayment.personalRepInPeriodOfAdmin.agent.title=Based on the details in this re
 
 repayment.yes.content=If you paid by debit card, we will attempt to refund it. If you paid by another method, you’ll be asked for your bank details on a later page.
 
-repayment.helpText=If you select ‘yes’ and are due a repayment of Capital Gains Tax, HMRC will repay you the money within 30 days. If you are not due a repayment you will not be contacted.
-repayment.agent.helpText=If you select ‘yes’ and your client is due a repayment of Capital Gains Tax, HMRC will repay them the money within 30 days. If they are not due a repayment they will not be contacted.
-repayment.trust.helpText=If you select ‘yes’ and the trust is due a repayment of Capital Gains Tax, HMRC will repay the money within 30 days. If it is not due a repayment you will not be contacted.
+repayment.helpText=If you select ‘yes’ and are due a repayment of Capital Gains Tax, HMRC will repay you the money. If you are not due a repayment, you will not be contacted.
+repayment.agent.helpText=If you select ‘yes’ and your client is due a repayment of Capital Gains Tax, HMRC will repay them the money. If they are not due a repayment they will not be contacted.
+repayment.trust.helpText=If you select ‘yes’ and the trust is due a repayment of Capital Gains Tax, HMRC will repay the money. If it is not due a repayment you will not be contacted.
```

Comment:
Uniformly add comma for all 3 messages
`If it is not due a repayment, you will not be contacted.`